be2-scala/src/test/scala/ch/epfl/pop/storage/DbActorSuite.scala

```diff
@@ -285,7 +285,7 @@ class DbActorSuite extends TestKit(ActorSystem("DbActorSuiteActorSystem")) with
     // arrange
     val messageRollCall: Message = MessageExample.MESSAGE_CLOSEROLLCALL
     val messageLao: Message = MessageExample.MESSAGE_CREATELAO_SIMPLIFIED
-    val address: String = "ws://popdemo.dedis.ch"
+    val address: Option[String] = Option("ws://popdemo.dedis.ch")
```

Comment:
Just as a detail, using Option works without issues, as it yields Some(x) for some value x and None for a null input, but using Some(x) directly is also a possibility and can be clearer in the code